modules/bitgo/example/js/btc/make-lightning-deposit.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Create lightning deposit address and deposit on-chain funds to a wallet's lightning balance.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
+
+const bitgo = new BitGoJS.BitGo({ env });
+
+// set your access token here
+const accessToken = '';
+
+// set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
+const walletId = '';
+
+async function makeLightningDeposit() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+  const wallet = await bitgo.coin(coin).wallets().get({ id: walletId });
+  const lightning = wallet.lightning();
+
+  const lightningDepositAddress = await lightning.createDepositAddress();
+  console.log('Lightning Deposit Address:', lightningDepositAddress.address);
+
+  const initialLightningBalance = await lightning.getBalance();
+  console.log('Initial Lightning Balance:', initialLightningBalance.balance);
+  const initialBalance = wallet.balance();
+  console.log('Initial Balance:', initialBalance);
+
+  const deposit = await lightning.deposit({ amount: 100000, address: lightningDepositAddress.address });
```

Comment:
I think we can simplify this a bit and not make the separate `createDepositAddress` call. If we don't pass an `address` to `lightning.deposit` it will make one for us.

As a side note, I'm actually thinking we should remove `address` as a param for `lightning.deposit` altogether. We're not checking that the provided address is actually a lightning address, and there's no easy way to check for that.